numba/parfors/parfor.py

```diff
@@ -1873,10 +1873,10 @@ def _type_getitem(self, args):
         return self.pass_states.typingctx.resolve_function_type(fnty, tuple(args), {})
 
 
-def _make_index_var(typemap, scope, index_vars, body_block):
+def _make_index_var(typemap, scope, index_vars, body_block, force_tuple=False):
```

Comment:
Might be a good idea to add a comment/docstring about what `force_tuple` is going to do and/or the situations where it is needed?